src/runtime/virtcontainers/pkg/annotations/annotations.go

```diff
@@ -37,6 +37,9 @@ const (
 	// ImagePath is a sandbox annotation for passing a per container path pointing at the guest image that will run in the container VM.
 	ImagePath = kataAnnotHypervisorPrefix + "image"
 
+	// ImagePath is a sandbox annotation for passing a per container path pointing at the guest image that will run in the container VM.
```

Comment:
nit: IGVM path not image path in comment. further, comment should be rephrased

Reply:
updated